### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by sangeetha-murugesan-sda9**
Added ReadMe Document , Added updated ScreenShots for likes and dislikes,Added Front-end

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -7,9 +7,10 @@
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
     "@varld/popover": "^0.1.0",
+    "particles-bg": "^2.5.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-modal-login": "^1.4.5",
+    "react-modal-login": "^1.3.3",
```
Comment (clecardona) [inline frontend/package.json]: could you correct that by keeping the last version **1.4.5**  or not commiting the package.json

File: LICENSE [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```
Comment (clecardona) [inline LICENSE]: i think we may keep the license. just copy it from the repo to your branch and commit again

File: frontend/src/pages/auth/AuthPage.jsx [hunk 3]
```diff
@@ -32,8 +32,8 @@ export default function LoginPage() {
     return (
         <div className="wrapper">
                     <div className="col-md-6 " style={{ color: "white" }}>
-                        <h1>InStyle</h1>
-                        <p></p>
+                        {/*<h1>InStyle</h1>*/}
+                        {/*<p></p>*/}
                     </div>
```
Comment (clecardona) [inline frontend/src/pages/auth/AuthPage.jsx]: i think you should keep the uncommented one

**Review by clecardona — CHANGES_REQUESTED** [review]
can you correct the comments then it is ok

**Review by clecardona — APPROVED** [review]

**Review by Ayatj — APPROVED** [review]
Approve Read Me

**Pull request merged**